lib/pymedphys/tests/gamma/test_gamma_shell.py

```diff
@@ -211,3 +211,55 @@ def test_distance_0_gives_1_point():
     )
 
     assert len(x) == 1 & len(y) == 1 & len(z) == 1
+
+
+def test_if_descending_coordinate_in_DICOM():
+    """Testing if the coordinate along an axis in DICOM is descending
+    0020,0037  Image Orientation (Patient):
+    head-in supine,  1\0\0\0\1\0  (all ascending in DICOM)
+    head-in prone,  -1\0\0\0\-1\0 (descending x, descending y in DICOM)
+    feet-in supine, -1\0\0\0\1\0  (descending x, descending z in DICOM)
+    feet-in prone,   1\0\0\0\-1\0 (descending y, descending z in DICOM)
```

Comment:
Is this always true? Can the `GridFrameOffsetVector` also be descending such that feet-first DICOM coordinates are ascending when read using `zyx_and_dose_from_dataset()` ?

Reply:
I don't know, but my current test could give the error
\interpolate.py", line 2475, in init
"ascending" % i)
ValueError: The points in dimension 0 must be strictly ascending